lib/src/main/java/com/auth0/android/lock/Auth0Parcelable.java

```diff
@@ -61,14 +62,10 @@ public void writeToParcel(@NonNull Parcel dest, int flags) {
         dest.writeString(auth0.getClientId());
         dest.writeString(auth0.getDomainUrl());
         dest.writeString(auth0.getConfigurationUrl());
-        dest.writeByte((byte) (auth0.isOIDCConformant() ? WITH_DATA : WITHOUT_DATA));
-        dest.writeByte((byte) (auth0.isLoggingEnabled() ? WITH_DATA : WITHOUT_DATA));
-        dest.writeByte((byte) (auth0.getTelemetry() != null ? WITH_DATA : WITHOUT_DATA));
-        if (auth0.getTelemetry() != null) {
-            dest.writeString(auth0.getTelemetry().getName());
-            dest.writeString(auth0.getTelemetry().getVersion());
-            dest.writeString(auth0.getTelemetry().getLibraryVersion());
-        }
+        //FIXME: Find a way to pass the NetworkingClient implementation
```

Comment:
Mostly just for my own education, but what does this comment mean?

Reply:
This relates to the "note" I added to the PR description. Customizing the networking client was made available in the latest release of the SDK. But the way we accept that client is with a setter in the `Auth0` class. 

This library has an implementation that makes `Auth0` "Parcelable" (similar to `Serializable`), so we could pass it across instances/classes to set up the widget. But because the `NetworkingClient` implementation cannot be serialized, it's not straightforward to update the `Auth0` class to support it. I added the comment to come back to this later and evaluate if this is something we need to backport today or not.

Reply:
Hello there! I realize this is a _very_ old PR, but I was wondering if this was ever floated to be fixed, as I noticed it's still in the current version...being able to pass in a custom networking client would be super useful for my team right now as we have to do weird proxy stuff but would prefer not to do a full migration to the raw SDK at the moment 😅 

My understanding is lock is not really receiving any new feature requests, but wondering if this would qualify as a bugfix and if we can contribute to a PR somehow!

Thanks so much!